externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/events/JsonEventLogger.scala

```diff
@@ -35,8 +35,10 @@ import org.apache.kyuubi.service.AbstractService
 
 /**
  * This event logger logs Kyuubi engine events in JSON file format.
- * The hierarchical directory structure is {ENGINE_EVENT_JSON_LOG_PATH}/{eventType}/{logName}.json
- * The {eventType} is based on core concepts of the Kyuubi systems, e.g. engine/session/statement
+ * The hierarchical directory structure is:
+ *   ${ENGINE_EVENT_JSON_LOG_PATH}/event=${eventType}/day=${date}/${logName}.json
```

Comment:
Maybe is more friendly : )
`${ENGINE_EVENT_JSON_LOG_PATH}/day=${date}/event=${eventType}/${logName}.json`

Reply:
Since the events have different schema, we can create view on `${ENGINE_EVENT_JSON_LOG_PATH}/event=${eventType}`